### PR TITLE
Added support for collectionFormat to array coercion

### PR DESCRIPTION
**Pull request by phillip-hall**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -128,3 +133,18 @@ function byLocation(location) {
 function getCoercer(type) {
   return COERCION_STRATEGIES[type];
 }
+
+function pathsep(format) {
+  switch (format) {
+    case 'csv':
```
Comment (jsdevel) [inline index.js:144]: Hmm.  I don't think these are valid item types for arrays.  See https://github.com/OAI/OpenAPI-Specification/blob/master/schemas/v2.0/schema.json#L1092.

Is it possible to handle this with middleware outside this project?

Comment (phillip-hall) [inline index.js:144]: The PR relates to the "collectionFormat" that defines the delimiter an array type parameter uses.  See [github.com/OAI/OpenAPI-Specification/blob/master/schemas/v2.0/schema.json#L1512](url) & [http://swagger.io/specification/#parameterObject](url)

Other swagger node modules coerce the parameter arrays from strings into actual arrays using the delimiter defined, or 'csv' as a default.

**Comment by jsdevel** [comment]
I see!  Cool!  Same thing here then with tests.  Can you add some?

**Comment by jsdevel** [comment]
Thanks @phillip-hall !

**Comment by jsdevel** [comment]
@phillip-hall would you mind submitting a PR against `express-openapi` with `express-openapi-coercion` bumped to `0.4.0` in `package.json` and updates to the README there?

**Comment by jsdevel** [comment]
Also, it would be great if you could update the README in this project as well to state that we now support formData.  Thanks again!

**Pull request merged**